js/src/util.js

```diff
@@ -84,7 +84,7 @@ const Util = (($) => {
       }
 
       const validSelector = selector
-      if (selector.charAt(0) === '#') {
+      if (selector.charAt(0) === '#' && selector.indexOf(',') === -1) {
```

Comment:
How about:

1. Spaces, e.g. `#my-id div`
2. Combinators like `+` and `>`, e.g. `#my-id>div`, `#my-id+div`, etc

Reply:
You're totally right @glebm I'll fix that asap thanks